File: javascript/1018.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
let valor = parseInt(input);

console.log(valor);

let TrocoItalo = parseInt(valor / 100);
console.log(`${TrocoItalo} nota(s) de R$ ${100},00`);
valor = valor % 100;


TrocoItalo = parseInt(valor / 50);
console.log(`${TrocoItalo} nota(s) de R$ ${50},00`);
valor = valor % 50;


TrocoItalo = parseInt(valor / 20);
console.log(`${TrocoItalo} nota(s) de R$ ${20},00`);
valor = valor % 20;


TrocoItalo = parseInt(valor / 10);
console.log(`${TrocoItalo} nota(s) de R$ ${10},00`);
valor = valor % 10;



TrocoItalo = parseInt(valor / 5);
console.log(`${TrocoItalo} nota(s) de R$ ${5},00`);
valor = valor % 5;


TrocoItalo = parseInt(valor / 2);
console.log(`${TrocoItalo} nota(s) de R$ ${2},00`);
valor = valor % 2;


TrocoItalo = parseInt(valor / 1);
console.log(`${TrocoItalo} nota(s) de R$ ${1},00`);
valor = valor % 1;

File: java/1018.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int valor = sc.nextInt();
        System.out.println(valor);

        int trocoItalo;

        trocoItalo = valor / 100;
        System.out.println(trocoItalo + " nota(s) de R$ 100,00");
        valor = valor % 100;

        trocoItalo = valor / 50;
        System.out.println(trocoItalo + " nota(s) de R$ 50,00");
        valor = valor % 50;

        trocoItalo = valor / 20;
        System.out.println(trocoItalo + " nota(s) de R$ 20,00");
        valor = valor % 20;

        trocoItalo = valor / 10;
        System.out.println(trocoItalo + " nota(s) de R$ 10,00");
        valor = valor % 10;

        trocoItalo = valor / 5;
        System.out.println(trocoItalo + " nota(s) de R$ 5,00");
        valor = valor % 5;

        trocoItalo = valor / 2;
        System.out.println(trocoItalo + " nota(s) de R$ 2,00");
        valor = valor % 2;

        trocoItalo = valor / 1;
        System.out.println(trocoItalo + " nota(s) de R$ 1,00");
        valor = valor % 1;

        sc.close();
    }
}
